configs/config.d/redis.conf

```diff
@@ -1,5 +1,6 @@
 <source>
   @type tail
+  # The format across lines is not consistent, can not deduce structure.
```

Comment:
🤦‍♀️  What's that like? I saw a blog post: http://build47.com/redis-log-format-levels/. Not sure how useful it would be though.

Reply:
Some log lines include the log level, and others don't. I'll find the example and paste it here.

Reply:
This is what the log file contains. WDYT?

```23439:C 30 Oct 15:33:59.003 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
23439:C 30 Oct 15:33:59.004 # Redis version=4.0.2, bits=64, commit=00000000, modified=0, pid=23439, just started
23439:C 30 Oct 15:33:59.004 # Configuration loaded
23440:M 30 Oct 15:33:59.006 * Increased maximum number of open files to 10032 (it was originally set to 1024).
                _._                                                  
           _.-``__ ''-._                                             
      _.-``    `.  `_.  ''-._           Redis 4.0.2 (00000000/0) 64 bit
  .-`` .-```.  ```\/    _.,_ ''-._                                   
 (    '      ,       .-`  | `,    )     Running in standalone mode
 |`-._`-...-` __...-.``-._|'` _.-'|     Port: 6379
 |    `-._   `._    /     _.-'    |     PID: 23440
  `-._    `-._  `-./  _.-'    _.-'                                   
 |`-._`-._    `-.__.-'    _.-'_.-'|                                  
 |    `-._`-._        _.-'_.-'    |           http://redis.io        
  `-._    `-._`-.__.-'_.-'    _.-'                                   
 |`-._`-._    `-.__.-'    _.-'_.-'|                                  
 |    `-._`-._        _.-'_.-'    |                                  
  `-._    `-._`-.__.-'_.-'    _.-'                                   
      `-._    `-.__.-'    _.-'                                       
          `-._        _.-'                                           
              `-.__.-'                                               

23440:M 30 Oct 15:33:59.007 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
23440:M 30 Oct 15:33:59.007 # Server initialized
23440:M 30 Oct 15:33:59.007 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
23440:M 30 Oct 15:33:59.007 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
23440:M 30 Oct 15:33:59.007 * Ready to accept connections
23440:M 30 Oct 15:49:00.022 * 1 changes in 900 seconds. Saving...
23440:M 30 Oct 15:49:00.023 * Background saving started by pid 12431
12431:C 30 Oct 15:49:00.027 * DB saved on disk
12431:C 30 Oct 15:49:00.027 * RDB: 6 MB of memory used by copy-on-write
23440:M 30 Oct 15:49:00.123 * Background saving terminated with success```